services/QuillLMS/client/app/bundles/Staff/components/evidence/llmPromptDatasets/individualDataset/__tests__/activateLLMPromptsSection.test.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { QueryClient, QueryClientProvider } from 'react-query';
+
+import ActivatedLLMPromptsSection from '../activatedLLMPromptsSection';
+import { ActivatedLLMPromptInterface } from '../../../../../interfaces/evidenceInterfaces';
+
+jest.mock('../../../../../utils/evidence/genAIAPIs', () => ({
+  fetchActivatedLLMPromptsForPrompt: jest.fn(),
+}));
+
+const queryClient = new QueryClient();
+
+describe('ActivatedLLMPromptsSection', () => {
+  const mockIndividualDatasetLinkBase = '/datasets';
+
+  const testDatasetId = 1;
+  const testPromptId = 123;
+  const testActiveActivatedPrompt: ActivatedLLMPromptInterface = {
+    id: 456,
+    active: true,
+    created_at: '2023-01-01T00:00:00Z',
+    updated_at: '2023-01-02T00:00:00Z',
+    llm_prompt_id: 789,
+    prompt_id: testPromptId,
+  };
+
+  const activatedPromptsResponse = {
+    activated_llm_prompts: [
+      {
+        id: 1,
+        notes: 'Test Notes',
+        trial: {
+          id: 10,
+          number: 5,
+          created_at: '2023-01-01T12:00:00Z',
+          optimal_correct: 90,
+          optimal_count: 100,
+          suboptimal_correct: 80,
+          suboptimal_count: 100,
+          average_g_eval_score: 85,
+          llm_version: '1.0.0',
+          llm_prompt: {
+            name: 'Prompt Name',
+            optimal_examples_count: 3,
+            suboptimal_examples_count: 2,
+            guidelines_count: 1,
+          },
+          dataset_id: testDatasetId,
+          dataset_version: '1.0',
+        },
+      },
+    ],
+  };
+
+  const sharedProps = {
+    promptId: testPromptId,
+    datasetId: testDatasetId,
+    individualDatasetLinkBase: mockIndividualDatasetLinkBase,
+    activeActivatedLLMPrompt: testActiveActivatedPrompt,
+
+  }
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+    queryClient.clear();
+  });
+
+  it('displays the table', async () => {
+    queryClient.setQueryData([`activated-llm-prompts-for-${testPromptId}`, testPromptId], {
+      data: activatedPromptsResponse,
+    });
+
+    const { asFragment } = render(
+      <QueryClientProvider client={queryClient}>
+        <ActivatedLLMPromptsSection {...sharedProps} />
+      </QueryClientProvider>
+    );
+
+    expect(screen.getByText(/Activated Prompts \(All Time\)/i)).toBeInTheDocument();
+    expect(screen.getByText(/Prompt Name/i)).toBeInTheDocument();
+    expect(screen.getByText(/Dataset 1.0/i)).toBeInTheDocument();
```

Comment:
Are these values hard-coded into the template, or are they based on some of the data defined in `activated_llm_prompts`?  If they're hard-coded, leaving this as-is makes sense, but if they're based on values in the input object, I think it would be clearer to have them referenced by variable name.  Such as
```javascript
expect(screen.getByText(/${activatedPromptsResponse. activated_llm_prompts[0]. trial.llm_prompt.name}/i)).toBeInTheDocument();
```
Or maybe something more readable, but something that makes it clear that the input should impact the output.